core/domain/exp_domain_test.py

```diff
@@ -529,7 +529,8 @@ def test_validation(self):
             'tagged_skill_misconception_id': None
         })
 
-        init_state.update_interaction_answer_groups(old_answer_groups)
+        init_state.update_interaction_answer_groups(
+            state_domain.AnswerGroup.from_dict(old_answer_groups))
```

Comment:
`old_answer_groups` is a list, right? If yes, then are we expecting `from_dict` to accept list?

Reply:
Yes. Changing it to `from_dicts`.